assets/app/views/browse/_pod-details.html

```diff
@@ -3,12 +3,20 @@
     <div class="col-lg-6">
       <h3>Status</h3>
       <dl class="dl-horizontal left">
-        <dt>Phase:</dt>
+        <dt>Status:</dt>
         <dd>
-          <status-icon status="pod.status.phase"></status-icon>
-          {{pod.status.phase}}
-          <span ng-if="pod.status.reason">
-            ({{pod.status.reason}})
+          <status-icon status="pod | podStatus"></status-icon>
+          {{pod | podStatus | sentenceCase}}
+          <span ng-if="pod.metadata.deletionTimestamp">(expires {{pod.metadata.deletionTimestamp | date : 'medium'}})</span>
```

Comment:
hmm I would have expected that the deletionTimestamp was the time it was marked for deletion, and the expiration would be the end of the grace period? @derekwaynecarr